apps/pipelines/models.py

```diff
@@ -379,19 +380,23 @@ def _save_message_to_history(
         return chat_message
 
     @transaction.atomic()
-    def create_new_version(self):
+    def create_new_version(self, copy_experiment: bool = False):
```

Comment:
Nitty comment: Can we rename the `copy_experiment` param to something like `is_copy`? My thought is that `copy_experiment` doesn't quite fit the description when this method is called on non-experiment models.